### PR TITLE
Remove references to old versions

### PR DESCRIPTION
**Pull request by btat**
Fixes https://github.com/rancher/rancher-docs/issues/725.

### CODE REVIEW

File: docs/how-to-guides/advanced-user-guides/configure-layer-7-nginx-load-balancer.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ docker run -d --restart=unless-stopped \
   rancher/rancher:latest
 ```
 
-As of Rancher v2.5, privileged access is [required.](../../pages-for-subheaders/rancher-on-a-single-node-with-docker.md#privileged-access-for-rancher)
+Note that privileged access is [required.](../../pages-for-subheaders/rancher-on-a-single-node-with-docker.md#privileged-access-for-rancher)
```
Comment (martyav) [inline docs/how-to-guides/advanced-user-guides/configure-layer-7-nginx-load-balancer.md:211]:
```suggestion
This operation requires [privileged access](../../pages-for-subheaders/rancher-on-a-single-node-with-docker.md#privileged-access-for-rancher).
```

There's a lot of passive voice in this page.

File: docs/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Rancher doesn't manage version control for cluster templates. Version control is
 
 ## RKE2 Cluster Template
 
-The example repository for an RKE2 cluster template is [here](https://github.com/rancher/cluster-template-examples). As of Rancher v2.6.0, we provide an RKE2 cluster template and may add more in the future.
+The example repository for an RKE2 cluster template is [here](https://github.com/rancher/cluster-template-examples).
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates.md:23]:
```suggestion
See the example repository for an [RKE2 cluster template](https://github.com/rancher/cluster-template-examples).
```

File: docs/integrations-in-rancher/harvester.md [hunk 3]
```diff
@@ -26,7 +20,7 @@ To navigate to the Harvester cluster, click **☰ > Virtualization Management**.
 
 ### Harvester Node Driver
 
-As of Rancher v2.6.3, the [Harvester node driver](https://docs.harvesterhci.io/v1.1/rancher/node/node-driver/) is GA for RKE and RKE2 options in Rancher. The node driver is available whether or not the Harvester feature flag is enabled. Note that the node driver is off by default. Users may create RKE or RKE2 clusters on Harvester only from the Cluster Management page.
+The [Harvester node driver](https://docs.harvesterhci.io/v1.1/rancher/node/node-driver/) is GA for RKE and RKE2 options in Rancher. The node driver is available whether or not the Harvester feature flag is enabled. Note that the node driver is off by default. Users may create RKE or RKE2 clusters on Harvester only from the Cluster Management page.
```
Comment (martyav) [inline docs/integrations-in-rancher/harvester.md:23]:
```suggestion
The [Harvester node driver](https://docs.harvesterhci.io/v1.1/rancher/node/node-driver/) is generally available for RKE and RKE2 options in Rancher. The node driver is available whether or not the Harvester feature flag is enabled. Note that the node driver is off by default. Users may create RKE or RKE2 clusters on Harvester only from the Cluster Management page.
```

File: docs/integrations-in-rancher/neuvector.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ title: NeuVector Integration
 
 ### NeuVector Integration in Rancher
 
-New in Rancher v2.6.5, [NeuVector 5.x](https://open-docs.neuvector.com/) is an open-source container-centric security platform that is now integrated into Rancher. NeuVector offers real-time compliance, visibility, and protection for critical applications and data during runtime. NeuVector provides a firewall, container process/file system monitoring, security auditing with CIS benchmarks, and vulnerability scanning. For more information on Rancher security, please see the [security documentation](../pages-for-subheaders/rancher-security.md).
+[NeuVector 5.x](https://open-docs.neuvector.com/) is an open-source container-centric security platform that is now integrated into Rancher. NeuVector offers real-time compliance, visibility, and protection for critical applications and data during runtime. NeuVector provides a firewall, container process/file system monitoring, security auditing with CIS benchmarks, and vulnerability scanning. For more information on Rancher security, please see the [security documentation](../pages-for-subheaders/rancher-security.md).
```
Comment (martyav) [inline docs/integrations-in-rancher/neuvector.md:7]:
```suggestion
[NeuVector 5.x](https://open-docs.neuvector.com/) is an open-source container-centric security platform that is integrated with Rancher. NeuVector offers real-time compliance, visibility, and protection for critical applications and data during runtime. NeuVector provides a firewall, container process/file system monitoring, security auditing with CIS benchmarks, and vulnerability scanning. For more information on Rancher security, please see the [security documentation](../pages-for-subheaders/rancher-security.md).
```

File: docs/pages-for-subheaders/cloud-marketplace.md [hunk 5]
```diff
@@ -4,4 +4,4 @@ title: Cloud Marketplace Integration
 
 Rancher offers integration with cloud marketplaces to easily purchase support for installations hosted on certain cloud providers. In addition, this integration also provides the ability to generate a supportconfig bundle which can be provided to rancher support.
 
-As of Rancher v2.6.7, only AWS is supported for this integration.
+Only AWS is supported for this integration.
```
Comment (martyav) [inline docs/pages-for-subheaders/cloud-marketplace.md:7]:
```suggestion
This integration only supports AWS.
```

File: docs/pages-for-subheaders/helm-charts-in-rancher.md [hunk 6]
```diff
@@ -4,18 +4,12 @@ title: Helm Charts in Rancher
 
 In this section, you'll learn how to manage Helm chart repositories and applications in Rancher. Helm chart repositories are managed using **Apps**. It uses a catalog-like system to import bundles of charts from repositories and then uses those charts to either deploy custom Helm applications or Rancher's tools such as Monitoring or Istio. Rancher tools come as pre-loaded repositories which deploy as standalone Helm charts. Any additional repositories are only added to the current cluster.
 
-### Changes in Rancher v2.6
+### Versioning Scheme
 
-Starting in Rancher v2.6.0, a new versioning scheme for Rancher feature charts was implemented. The changes are centered around the major version of the charts and the +up annotation for upstream charts, where applicable.
+The Rancher feature charts versioning scheme is centered around the major version of the charts and the +up annotation for upstream charts, where applicable.
```
Comment (martyav) [inline docs/pages-for-subheaders/helm-charts-in-rancher.md:9]:
```suggestion
The Rancher feature charts versioning scheme is centered around the major version of the charts and the +up annotation for upstream charts, where applicable.
```

+up? Should that get special formatting?

File: docs/pages-for-subheaders/helm-charts-in-rancher.md [hunk 7]
```diff
@@ -103,10 +97,9 @@ To add a private CA for Helm Chart repositories:
     [...]
     ```
 
-
 :::note Helm chart repositories with authentication
 
-As of Rancher v2.6.3, a new value `disableSameOriginCheck` has been added to the Repo.Spec. This allows users to bypass the same origin checks, sending the repository Authentication information as a Basic Auth Header with all API calls. This is not recommended but can be used as a temporary solution in cases of non-standard Helm chart repositories such as those that have redirects to a different origin URL.
+The Repo.Spec contains a `disableSameOriginCheck` that allows users to bypass the same origin checks, sending the repository Authentication information as a Basic Auth Header with all API calls. This is not recommended but can be used as a temporary solution in cases of non-standard Helm chart repositories such as those that have redirects to a different origin URL.
```
Comment (martyav) [inline docs/pages-for-subheaders/helm-charts-in-rancher.md:102]:
```suggestion
The Repo.Spec contains a `disableSameOriginCheck` value that allows users to bypass the same origin checks, sending the repository Authentication information as a Basic Auth Header with all API calls. This is not recommended but can be used as a temporary solution in cases of non-standard Helm chart repositories such as those that have redirects to a different origin URL.
```

File: docs/reference-guides/monitoring-v2-configuration/routes.md [hunk 8]
```diff
@@ -40,37 +40,13 @@ The route needs to refer to a [receiver](./receivers.md) that has already been c
...
-
-</TabItem>
-<TabItem value="Rancher before v2.6.5">
-
 | Field |    Default | Description |
 |-------|--------------|---------|
-| Group By |  N/a | The labels by which incoming alerts are grouped together. For example, `[ group_by: '[' <labelname>, ... ']' ]` Multiple alerts coming in for labels such as `cluster=A` and `alertname=LatencyHigh` can be batched into a single group. To aggregate by all possible labels, use the special value `'...'` as the sole label name, for example: `group_by: ['...']`  Grouping by `...` effectively disables aggregation entirely, passing through all alerts as-is. This is unlikely to be what you want, unless you have a very low alert volume or your upstream notification system performs its own grouping. |
+| Group By |  N/a | List of labels to group by. Labels must not be repeated (unique list). Special label "..." (aggregate by all possible labels), if provided, must be the only element in the list. Accepts a list of strings. See the [upstream documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#route) for details. |
```
Comment (martyav) [inline docs/reference-guides/monitoring-v2-configuration/routes.md:45]:
```suggestion
| Group By |  N/a | List of labels to group by. All labels must be unique. Special label "..." (aggregate by all possible labels), if provided, must be the only element in the list. Accepts a list of strings. See the [upstream documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#route) for details. |
```

File: versioned_docs/version-2.7/getting-started/installation-and-upgrade/upgrade-and-roll-back-kubernetes.md [hunk 9]
```diff
@@ -96,7 +96,7 @@ To enable draining each node during a cluster upgrade,
 
 :::note
 
-As of Rancher v2.4.0, there is a [known issue](https://github.com/rancher/rancher/issues/25478) in which the Rancher UI doesn't show state of etcd and controlplane as drained, even though they are being drained.
+There is a [known issue](https://github.com/rancher/rancher/issues/25478) in which the Rancher UI doesn't show state of etcd and controlplane as drained, even though they are being drained.
```
Comment (martyav) [inline versioned_docs/version-2.7/getting-started/installation-and-upgrade/upgrade-and-roll-back-kubernetes.md:99]:
```suggestion
There is a [known issue](https://github.com/rancher/rancher/issues/25478) in which the Rancher UI doesn't show the state of etcd and controlplane as drained, even though they are being drained.
```

File: versioned_docs/version-2.7/how-to-guides/advanced-user-guides/configure-layer-7-nginx-load-balancer.md [hunk 10]
```diff
@@ -208,7 +208,7 @@ docker run -d --restart=unless-stopped \
   rancher/rancher:latest
 ```
 
-As of Rancher v2.5, privileged access is [required.](../../pages-for-subheaders/rancher-on-a-single-node-with-docker.md#privileged-access-for-rancher)
+Note that privileged access is [required.](../../pages-for-subheaders/rancher-on-a-single-node-with-docker.md#privileged-access-for-rancher)
```
Comment (martyav) [inline versioned_docs/version-2.7/how-to-guides/advanced-user-guides/configure-layer-7-nginx-load-balancer.md:211]:
```suggestion
This operation requires [privileged access](../../pages-for-subheaders/rancher-on-a-single-node-with-docker.md#privileged-access-for-rancher).
```

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates.md [hunk 11]
```diff
@@ -20,7 +20,7 @@ Rancher doesn't manage version control for cluster templates. Version control is
 
 ## RKE2 Cluster Template
 
-The example repository for an RKE2 cluster template is [here](https://github.com/rancher/cluster-template-examples). As of Rancher v2.6.0, we provide an RKE2 cluster template and may add more in the future.
+The example repository for an RKE2 cluster template is [here](https://github.com/rancher/cluster-template-examples).
```
Comment (martyav) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates.md:23]: See comment above on matching page

File: versioned_docs/version-2.7/reference-guides/monitoring-v2-configuration/routes.md [hunk 12]
```diff
@@ -40,37 +40,13 @@ The route needs to refer to a [receiver](./receivers.md) that has already been c
...
-
-</TabItem>
-<TabItem value="Rancher before v2.6.5">
-
 | Field |    Default | Description |
 |-------|--------------|---------|
-| Group By |  N/a | The labels by which incoming alerts are grouped together. For example, `[ group_by: '[' <labelname>, ... ']' ]` Multiple alerts coming in for labels such as `cluster=A` and `alertname=LatencyHigh` can be batched into a single group. To aggregate by all possible labels, use the special value `'...'` as the sole label name, for example: `group_by: ['...']`  Grouping by `...` effectively disables aggregation entirely, passing through all alerts as-is. This is unlikely to be what you want, unless you have a very low alert volume or your upstream notification system performs its own grouping. |
+| Group By |  N/a | List of labels to group by. Labels must not be repeated (unique list). Special label "..." (aggregate by all possible labels), if provided, must be the only element in the list. Accepts a list of strings. See the [upstream documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#route) for details. |
```
Comment (martyav) [inline versioned_docs/version-2.7/reference-guides/monitoring-v2-configuration/routes.md:45]:
```suggestion
| Group By |  N/a | List of labels to group by. Labels must be unique. Special label "..." (aggregate by all possible labels), if provided, must be the only element in the list. Accepts a list of strings. See the [upstream documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#route) for details. |
```

**Review by martyav — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates.md (hunk 11 above)
Comment (btat) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/manage-clusters/manage-cluster-templates.md:23]:
```suggestion
See the example repository for an [RKE2 cluster template](https://github.com/rancher/cluster-template-examples).
```

File: versioned_docs/version-2.7/integrations-in-rancher/harvester.md [hunk 13]
```diff
@@ -26,7 +20,7 @@ To navigate to the Harvester cluster, click **☰ > Virtualization Management**.
 
 ### Harvester Node Driver
 
-As of Rancher v2.6.3, the [Harvester node driver](https://docs.harvesterhci.io/v1.1/rancher/node/node-driver/) is GA for RKE and RKE2 options in Rancher. The node driver is available whether or not the Harvester feature flag is enabled. Note that the node driver is off by default. Users may create RKE or RKE2 clusters on Harvester only from the Cluster Management page.
+The [Harvester node driver](https://docs.harvesterhci.io/v1.1/rancher/node/node-driver/) is GA for RKE and RKE2 options in Rancher. The node driver is available whether or not the Harvester feature flag is enabled. Note that the node driver is off by default. Users may create RKE or RKE2 clusters on Harvester only from the Cluster Management page.
```
Comment (btat) [inline versioned_docs/version-2.7/integrations-in-rancher/harvester.md:23]:
```suggestion
The [Harvester node driver](https://docs.harvesterhci.io/v1.1/rancher/node/node-driver/) is generally available for RKE and RKE2 options in Rancher. The node driver is available whether or not the Harvester feature flag is enabled. Note that the node driver is off by default. Users may create RKE or RKE2 clusters on Harvester only from the Cluster Management page.
```

File: versioned_docs/version-2.7/integrations-in-rancher/neuvector.md [hunk 14]
```diff
@@ -4,7 +4,7 @@ title: NeuVector Integration
 
 ### NeuVector Integration in Rancher
 
-New in Rancher v2.6.5, [NeuVector 5.x](https://open-docs.neuvector.com/) is an open-source container-centric security platform that is now integrated into Rancher. NeuVector offers real-time compliance, visibility, and protection for critical applications and data during runtime. NeuVector provides a firewall, container process/file system monitoring, security auditing with CIS benchmarks, and vulnerability scanning. For more information on Rancher security, please see the [security documentation](../pages-for-subheaders/rancher-security.md).
+[NeuVector 5.x](https://open-docs.neuvector.com/) is an open-source container-centric security platform that is now integrated into Rancher. NeuVector offers real-time compliance, visibility, and protection for critical applications and data during runtime. NeuVector provides a firewall, container process/file system monitoring, security auditing with CIS benchmarks, and vulnerability scanning. For more information on Rancher security, please see the [security documentation](../pages-for-subheaders/rancher-security.md).
```
Comment (btat) [inline versioned_docs/version-2.7/integrations-in-rancher/neuvector.md:7]:
```suggestion
[NeuVector 5.x](https://open-docs.neuvector.com/) is an open-source container-centric security platform that is integrated with Rancher. NeuVector offers real-time compliance, visibility, and protection for critical applications and data during runtime. NeuVector provides a firewall, container process/file system monitoring, security auditing with CIS benchmarks, and vulnerability scanning. For more information on Rancher security, please see the [security documentation](../pages-for-subheaders/rancher-security.md).
```

File: versioned_docs/version-2.7/pages-for-subheaders/cloud-marketplace.md [hunk 15]
```diff
@@ -4,4 +4,4 @@ title: Cloud Marketplace Integration
 
 Rancher offers integration with cloud marketplaces to easily purchase support for installations hosted on certain cloud providers. In addition, this integration also provides the ability to generate a supportconfig bundle which can be provided to rancher support.
 
-As of Rancher v2.6.7, only AWS is supported for this integration.
+Only AWS is supported for this integration.
```
Comment (btat) [inline versioned_docs/version-2.7/pages-for-subheaders/cloud-marketplace.md:7]:
```suggestion
This integration only supports AWS.
```

File: versioned_docs/version-2.7/pages-for-subheaders/helm-charts-in-rancher.md [hunk 16]
```diff
@@ -4,18 +4,12 @@ title: Helm Charts in Rancher
 
 In this section, you'll learn how to manage Helm chart repositories and applications in Rancher. Helm chart repositories are managed using **Apps**. It uses a catalog-like system to import bundles of charts from repositories and then uses those charts to either deploy custom Helm applications or Rancher's tools such as Monitoring or Istio. Rancher tools come as pre-loaded repositories which deploy as standalone Helm charts. Any additional repositories are only added to the current cluster.
 
-### Changes in Rancher v2.6
+### Versioning Scheme
 
-Starting in Rancher v2.6.0, a new versioning scheme for Rancher feature charts was implemented. The changes are centered around the major version of the charts and the +up annotation for upstream charts, where applicable.
+The Rancher feature charts versioning scheme is centered around the major version of the charts and the +up annotation for upstream charts, where applicable.
```
Comment (btat) [inline versioned_docs/version-2.7/pages-for-subheaders/helm-charts-in-rancher.md:9]:
```suggestion
The Rancher feature charts versioning scheme is centered around the major version of the charts and the `+up` annotation for upstream charts, where applicable.
```

**Comment by btat** [comment]
@martyav feedback has been applied (and added to versioned page where applicable).

**Pull request merged**